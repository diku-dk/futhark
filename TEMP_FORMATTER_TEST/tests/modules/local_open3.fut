-- Local open with nested modules, defined elsewhere.
-- ==

module has_x = {
  def x = 1i32
}

module has_has_x = {
  module has_x = has_x
}

def main = has_has_x.(has_x.x)
