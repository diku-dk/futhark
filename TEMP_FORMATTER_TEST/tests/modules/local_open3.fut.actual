a0f9122c4270dc6fd3dc935b8ce57b1c
