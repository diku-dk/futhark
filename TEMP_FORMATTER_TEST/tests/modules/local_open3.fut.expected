5d9747773983b040d3d0dfca4c45bebd
